k.makoyid/oop_code/Visualizer.py

```diff
@@ -0,0 +1,26 @@
+import matplotlib.pyplot as plt
+
+
+class Visualizer:
+    @staticmethod
+    def plot_results(train_losses, train_accuracies, test_losses, test_accuracies):
+        plt.figure(figsize=(12, 8))
+
+        plt.subplot(221)
+        plt.plot(train_losses, label='Train')
+        plt.plot(test_losses, label='Test')
+        plt.xlabel('Iterations (x10)')
+        plt.ylabel('Loss')
+        plt.title('Loss vs Iterations')
+        plt.legend()
+
+        plt.subplot(222)
+        plt.plot(train_accuracies, label='Train')
+        plt.plot(test_accuracies, label='Test')
+        plt.xlabel('Iterations (x10)')

```

Comment:
Will there always be x10 iterations? It's better to get this value from as a function argument